otterdog/eclipse-zenoh.jsonnet

```diff
@@ -281,173 +289,183 @@ orgs.newOrg('eclipse-zenoh') {
+        customRuleset('main'),
+      ],
     },
     orgs.newRepo('zenoh-plugin-dds') {
       allow_auto_merge: true,
       allow_merge_commit: true,
       allow_update_branch: false,
       delete_branch_on_merge: false,
-      description: "A zenoh plug-in that allows to transparently route DDS data.  This plugin can be used by DDS applications, such as ROS2 robotic applications and others, to leverage the zenoh for geographical routing or for better scaling discovery.",
-      homepage: "",
+      dependabot_security_updates_enabled: true,
+      description: 'A zenoh plug-in that allows to transparently route DDS data.  This plugin can be used by DDS applications, such as ROS2 robotic applications and others, to leverage the zenoh for geographical routing or for better scaling discovery.',
+      homepage: '',
       topics+: [
-        "cyclonedds",
-        "dds",
-        "edge-computing",
-        "robotics",
-        "ros2",
-        "zenoh"
+        'cyclonedds',
+        'dds',
+        'edge-computing',
+        'robotics',
+        'ros2',
+        'zenoh',
       ],
       web_commit_signoff_required: false,
       workflows+: {
-        default_workflow_permissions: "write",
+        default_workflow_permissions: 'write',
       },
       rulesets: [
-        customRuleset("main"),
+        customRuleset('main'),
       ],
     },
     orgs.newRepo('zenoh-plugin-mqtt') {
       allow_auto_merge: true,
       allow_merge_commit: true,
       allow_update_branch: false,
       delete_branch_on_merge: false,
-      description: "A Zenoh plug-in that allows to integrate and/or route MQTT pub/sub with Eclipse Zenoh.",
+      dependabot_security_updates_enabled: true,
+      description: 'A Zenoh plug-in that allows to integrate and/or route MQTT pub/sub with Eclipse Zenoh.',
       web_commit_signoff_required: false,
       workflows+: {
-        default_workflow_permissions: "write",
+        default_workflow_permissions: 'write',
       },
       rulesets: [
-        customRuleset("main"),
+        customRuleset('main'),
       ],
     },
     orgs.newRepo('zenoh-plugin-ros1') {
       allow_auto_merge: true,
       allow_merge_commit: true,
       allow_update_branch: false,
       delete_branch_on_merge: false,
+      dependabot_security_updates_enabled: true,
       web_commit_signoff_required: false,
       workflows+: {
-        default_workflow_permissions: "write",
+        default_workflow_permissions: 'write',
       },
       rulesets: [
-        customRuleset("main"),
+        customRuleset('main'),
       ],
     },
     orgs.newRepo('zenoh-plugin-ros2dds') {
       allow_auto_merge: true,
       allow_merge_commit: true,
       allow_update_branch: false,
       delete_branch_on_merge: false,
-      description: "A Zenoh plug-in for ROS2 with a DDS RMW.",
-      homepage: "https://zenoh.io",
+      dependabot_security_updates_enabled: true,
+      description: 'A Zenoh plug-in for ROS2 with a DDS RMW.',
```

Comment:
We should point to the RMW alternate report to explain the advantages of using Zenoh ROS2 plugin.  @JEnoch should  update this.

Reply:
You mean this post: https://discourse.ros.org/t/ros-2-alternative-middleware-report/ ? How about:

```
A Zenoh plug-in for ROS2 with a DDS RMW. See https://discourse.ros.org/t/ros-2-alternative-middleware-report/  for the advantages of using this plugin over other DDS RMW implementations.
```